contributing/code/bc.rst

```diff
@@ -32,6 +32,14 @@ The second section, "Working on Symfony Code", is targeted at Symfony
 contributors. This section lists detailed rules that every contributor needs to
 follow to ensure smooth upgrades for our users.
 
+.. warning::
+
+    Code marked with the ``@experimental`` or the ``@internal`` tags are
+    excluded from our Backwards Compatibility promise.
+
+    Also note that backward compatibility breaks are tolerated if they help fix
+    a security issue.
```

Comment:
if they help fix a security issue => if they are required to fix a security issue?